sound/soc/sof/ipc4-topology.c

```diff
@@ -2414,8 +2414,9 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 		msg->extension &= ~SOF_IPC4_MOD_EXT_PPL_ID_MASK;
 		msg->extension |= SOF_IPC4_MOD_EXT_PPL_ID(pipe_widget->instance_id);
 	}
-	dev_dbg(sdev->dev, "Create widget %s instance %d - pipe %d - core %d\n",
-		swidget->widget->name, swidget->instance_id, swidget->pipeline_id, swidget->core);
+	dev_dbg(sdev->dev, "Create widget %s instance %d - pipe %d - core %d - widget ID %#x\n",
+		swidget->widget->name, swidget->instance_id, swidget->pipeline_id,
+		swidget->core, msg->primary);
```

Comment:
`msg->primary` is _not_ widget ID, it is the IPC message's primary part. If you want to print out the ID of the widget, then it is better to print that out?